homeassistant/components/tilt_pi/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "tilt_pi",
```

Comment:
since we already have the BLE one, we should create a brand for this

Reply:
Apologies if I'm not understanding, but what would that concretely look like?

Right now the PR I have open in the brands repo adds the icons under `core_integrations/tilt_pi`.

The `tilt_ble` integration icons live in `core_integrations/tilt_ble`.

Would the result of creating a unified brand for this be icons existing in `core_brands/tilt`? And what does that mean for the change in this manifest file? Does it require a change in the tilt_ble component, and if so should that particular change be in a separate PR?

Reply:
Okay so first the brands concept in core, there's a folder with json files that is called brands and we can create one for tilt pi. This way if you search for tilt pi you automatically get to pick between the two integrations. 

That brand also needs assets but your assets are likely similar to the BLE one. So you move all your assets to the core_brands and then symlink those to core_integrations tilt_pi and ble

Reply:
I have updated https://github.com/home-assistant/brands/pull/6667 to leverage a new `core_brands/tilt` symlinked to the `tilt_pi` and `tilt_ble` core integrations.

Reply:
But then this PR still needs a new json file for the brand. If you do a find in the codebase for google.json you can find an example and I think once you see that you'll get what I try to explain :)